src/ui/DialogSection.ts

```diff
@@ -2,9 +2,11 @@ import Navigation from './DialogNavigation'
 
 export default abstract class Section {
    private element: JQuery;
+   private button: JQuery;
+   private isCollapsible : Boolean;
 
-   constructor(protected navigation: Navigation, private title?: string) {
-
+   constructor(protected navigation: Navigation, protected collapsible?: Boolean, private title?: string) {
```

Comment:
`Boolean` is a class, the correct type would be `boolean`.

Reply:
changed..